obspy/io/sac/LICENSE.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2015. Los Alamos National Security, LLC for pysac LA-CC-15-051. This
```

Comment:
As long as we can keep distributing all of ObsPy under LGPL I am fine with it. Could you not just call it `obspy.io.sac (originally based on pysac)` or something along these lines?